togetherly/lib/views/widgets/edit_chore_dialog.dart

```diff
@@ -46,19 +47,22 @@ class _EditChoreDialogState extends State<EditChoreDialog> {
       familyId: 0,
       name: 'Emma',
       icon: ProfileIcon.bear,
+      isParent: false,
```

Comment:
Any reason why we still have fake people here?

Reply:
Only because this is referencing the list of children currently assigned to a chore, and that hasn't been implemented yet.

Reply:
Yeah I suppose that would be fine, I'll push that change